_includes/social.html

```diff
@@ -0,0 +1,13 @@
+<ul class="social-media-list">
```

Comment:
The problem with this idea though is it doesn't validate the hash keys. Any key defined under the `site_hash` would be passed to the for block even if Minima doesn't have the svg icon for it..